src/deps.ts

```diff
@@ -22,6 +22,7 @@ export const deps = {
   get styledHeader(): typeof styledHeader.default { return fetch('./styled/header').default },
   get styledJSON(): typeof styledJSON.default { return fetch('./styled/json').default },
   get table(): typeof table.default { return fetch('./styled/table').default },
+  get wait(): typeof table.default { return fetch('./wait').default },
```

Comment:
this type is wrong. I think you might be able to use the new `import()` syntax to make this better in typescript 2.9 too